src/generator/AutoRest.Go/Extensions.cs

```diff
@@ -904,9 +904,9 @@ public static bool IsBodyParameter(this IParameter p)
         /// <returns></returns>
         public static string GetConstraint(string name, string constraintName, string constraintValue, bool chain = false)
         {
-            var value = constraintName == Constraint.Pattern.ToString()
-                                             ? string.Format("\"{0}\"", constraintValue.Replace("\\", "\\\\"))
-                                             : constraintValue;
+            var value = constraintName == Constraint.Pattern.ToString()  
```

Comment:
sounds good!

---
In reply to: [77410684](https://github.com/Azure/autorest/pull/1397#discussion_r77410684) [](ancestors = 77410684)